frontend/packages/console-app/src/components/file-upload/file-upload-context.ts

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { AlertVariant } from '@patternfly/react-core';
+import { useToast } from '@console/shared/src/components/toast';
+import {
+  ResolvedFileUpload,
+  isFileUpload,
+} from '@console/dynamic-plugin-sdk/src/extensions/file-upload';
+import { useActiveNamespace } from '@console/shared/src/hooks/useActiveNamespace';
+import { useResolvedExtensions } from '@console/dynamic-plugin-sdk/src/api/useResolvedExtensions';
```

Comment:
```suggestion
import {
  useResolvedExtensions,
  ResolvedFileUpload,
  isFileUpload,
} from '@console/dynamic-plugin-sdk';
import { useActiveNamespace } from '@console/shared';
```

Reply:
`import { useActiveNamespace } from '@console/shared';` this is done deliberately to avoid the below error
![import issue](https://user-images.githubusercontent.com/22490998/111736340-a8532480-88a3-11eb-860d-ceb41b0841d5.png)